lib/commands/context.js

```diff
@@ -1,4 +1,4 @@
-import { errors } from 'appium-base-driver';
+import { errors } from 'mobile-json-wire-protocol';
```

Comment:
Check with Simon. This got moved to the appium-base-driver in a more recent Appium Server version.